### DISCUSSION TITLE
Change name when configuring SMTP

### DISCUSSION THREAD

**Discussion by JayP718** [Budibase/budibase#5610] [Ideas]
Hi,

Some other low code applications allow the application to show a name when sending an SMTP email so instead of just the email which sometimes has to include the server name to be allowed through smtp, it will show a name more familiar to the end users. It would be great if we can add that name when configuring SMTP.

![image](https://user-images.githubusercontent.com/42209052/165509112-f1208060-9038-4512-baaf-d40754bc6f3c.png)

Jay

**Comment by melohagan** [comment]
Hey @JayP718 

Have you tried entering the name into the *From email address* field like in this SO post? https://stackoverflow.com/questions/28566873/how-to-add-a-display-name-to-mail-sent-via-smtp

Assuming that doesn't work, this is a nice feature request.

**Reply by JayP718** [reply]
Unfortunately this error appears :/ Thanks for recommendation though

![image](https://user-images.githubusercontent.com/42209052/166144654-a97e42ed-2ccf-42cb-bcaa-bb6d1c0f4529.png)

**Reply by shogunpurple** [reply]
Yep, that's going to happen because our backend checks for a valid email - which the above wouldn't be. I'm going to create an issue from this discussion to enable support for the name field
